comp/api/api/apiimpl/internal/agent/agent.go

```diff
@@ -208,11 +210,13 @@ func componentStatusHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func getStatus(w http.ResponseWriter, r *http.Request, statusComponent status.Component) {
+func getStatus(w http.ResponseWriter, r *http.Request, statusComponent status.Component, section optional.Option[string]) {
 	log.Info("Got a request for the status. Making status.")
 	verbose := r.URL.Query().Get("verbose") == "true"
 	format := r.URL.Query().Get("format")
 	var contentType string
+	var err error
```

Comment:
**🥜 nitpick:** I *believe* moving these declaration just above the if/else block (where they are used) might improve readability‏